content/pages/home.mdx

```diff
@@ -73,7 +73,7 @@ beforeBody:
         color: darkgray
         imgSrc: /images/service-cards/Expert.png
       - title: Rules
-        link: 'https://www.ssw.com.au/rules'
+        link: 'https://ssw.com.au/rules/'
```

Comment:
I think its better that we change the component to use <a> instead of <Link>
Theres no benefit of `<Link>` for external links

Reply:
I changed small cards to use a regular anchor tag, instead of <Link>, and reverted the Rules link.

Those 4 small cards will no longer prefetch. any other cards will not be affected